src/Dependencies/Collections/Extensions/IEnumerableExtensions.cs

```diff
@@ -69,7 +69,7 @@ public static ImmutableArray<T> ToImmutableArrayOrEmpty<T>(this IEnumerable<T>?
         {
             if (items == null)
             {
-                return ImmutableArray.Create<T>();
+                return [];
```

Comment:
Although I don't normally mind the change, if this is a source package does this create a problem for any of our consumers that aren't on a new enough compiler?